src/System/Information/Memory.hs

```diff
@@ -10,23 +10,25 @@ data MemoryInfo = MemoryInfo { memoryTotal :: Double
                              , memoryFree :: Double
                              , memoryBuffer :: Double
                              , memoryCache :: Double
+                             , memoryAvailable :: Double -- An estimate of how much memory is available for starting new applicaitons
```

Comment:
typo, applicaitons

Reply:
@willprice can you fix this and the merge conflicts?